arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -209,7 +209,7 @@ def validateXbrlFinally(val, *args, **kwargs):
     if val.authority == "UK":
         if modelXbrl.fileSource and modelXbrl.fileSource.taxonomyPackage and modelXbrl.fileSource.taxonomyPackage["publisherCountry"] != "GB":
             modelXbrl.error("UKFRC.1.2.publisherCountrySetting",
-                        _("The “Publisher Country” element of the report package metadata for a UKSEF report MUST be set to \"GB\" but was \"%(publisherCountry)s\"."), 
+                        _("The \"Publisher Country\" element of the report package metadata for a UKSEF report MUST be set to \"GB\" but was \"%(publisherCountry)s\"."), 
```

Comment:
this is fine, I usually go for and prefer `'string'` because of this problem here. Then you can just do `'The string "is quoted"'`